deploy/v1.8.0/avalanche.ts

```diff
@@ -0,0 +1,94 @@
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { dim, green } from '../../src/colors';
+import { PRIZE_DISTRIBUTION_FACTORY_MINIMUM_PICK_COST } from '../../src/constants';
+import { deployAndLog } from '../../src/deployAndLog';
+import { setManager } from '../../src/setManager';
+import { transferOwnership } from '../../src/transferOwnership';
+
+/**
+ * Prepares a PoolTogether Prize Pool Network to update from using PrizeTierHistory
+ * and PrizeDistributionFactory to using PrizeTierHistoryV2 and PrizeDistributionFactoryV2.
+ *
+ * NOTE: The final step to complete the update is a transition of the manager role on a PrizeDistributionBuffer to be the newly deployed PrizeDistributionFactoryV2.
+ */
+export default async function deployToAvalancheMainnet(hre: HardhatRuntimeEnvironment) {
+  if (process.env.DEPLOY === 'v1.8.0.avalanche') {
+    dim(`Deploying: PrizeTierHistoryV2 and PrizeDistributionFactoryV2 on Avalanche Mainnet`);
+    dim(`Version: 1.8.0`);
+  } else {
+    return;
+  }
+
+  const { getNamedAccounts, ethers, getChainId } = hre;
+
+  const { deployer, executiveTeam, defenderRelayer } = await getNamedAccounts();
+
+  const chainId = parseInt(await getChainId(), 10);
+
+  dim(`chainId ${chainId} `);
+  dim(`---------------------------------------------------`);
+  dim(`Named Accounts`);
+  dim(`---------------------------------------------------`);
+  dim(`deployer: ${deployer}`);
+  dim(`executiveTeam: ${executiveTeam}`);
+  dim(`defenderRelayer: ${defenderRelayer}`);
+  dim(`---------------------------------------------------\n`);
+  const startingBalance = await ethers.provider.getBalance((await ethers.getSigners())[0].address);
+
+  // ===================================================
+  // Deploy Contracts
+  // ===================================================
+
+  // Load existing contracts
+  const ticket = await hre.deployments.get('Ticket');
+  const pdb = await hre.deployments.get('PrizeDistributionBuffer');
```

Comment:
Rename with the full name:
```suggestion
  const prizeDistributionBuffer = await hre.deployments.get('PrizeDistributionBuffer');
```